test/EnvMetadata.test.tsx

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { render } from "@testing-library/react";
 import "@testing-library/jest-dom/extend-expect";
 import { EnvMetadata } from "../src/features/metadata/components/EnvMetadata";
-import { buildMapper } from "../src/utils/helpers/buildMapper";
 import { mockTheme } from "./testutils";
+// import { useGetEnviromentsQuery } from "../src/features/metadata";
```

Comment:
If we don't need this comment here then it should be removed

Reply:
Removed!